### PR TITLE
Add instructions to readme for accessing UK/Intl versions locally

### LINKED ISSUE #1736: Localhost to show International version
**Is your feature request related to a problem? Please describe.**
When on localhost, you can only see the UK variant of the page.

Due to the way the [`getOriginContext`](https://github.com/bbc/simorgh/blob/af741b24b53c3a9e40385bc50509424437fd3ee0/src/app/contexts/RequestContext/getOriginContext.js#L15-L17) logic is defined, when on localhost, `isUK` is always `true`, even if you're viewing the page outside of the UK (or via VPN). 

This makes it difficult for testers to test PRs for the international variants locally, on their machines. 

We should enable our application to show the International variant.

**For testing**
UK version of cookie banner:
Screen Shot 2019-05-17 at 15 56 34

International version of cookie banner:
Screen Shot 2019-05-17 at 16 15 22

**Describe the solution you'd like**
We could update `getOriginContext.js` to show the international variant locally. 

**Describe alternatives you've considered**
N/A

**Testing notes**
[Tester to complete]

Dev insight: Unit tests and testing locally should be sufficient. 

**Additional context**

### PR DESCRIPTION
**Pull request by rossgaskell**
Resolves #1736 

**Overall change:** Add instructions to readme for accessing UK/Intl versions locally

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Tests added for new features
- [ ] Test engineer approval
- [ ] I have followed [the merging checklist](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/MERGE_PROCESS.md) and this is ready to merge.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -77,6 +77,13 @@ To run LIVE bundles on localhost:
 - Then run `rm -rf build && npm run build:live && npm run start`
 - Visit a live article: http://localhost:7080/news/articles/c8xxl4l3dzeo
 
+## Changing request location
+
+Some features perform differently dependant on whether a user is located within the UK or internationally. You can explicitly request a specific version by accessing Simorgh via a specific localhost BBC domain:
+
```
Comment (sareh) [inline README.md:83]: For this to work, I believe we'd need to add a hosts file entry (`/etc/hosts` or `C:\Windows\System32\drivers\etc\hosts`)

```
127.0.0.1 localhost.bbc.co.uk
```

File: README.md [hunk 2]
```diff
@@ -77,6 +77,13 @@ To run LIVE bundles on localhost:
...
+- UK version: [http://localhost.bbc.co.uk:7080](http://localhost.bbc.co.uk:7080)
```
Comment (sareh) [inline README.md]: Linking to article example would ensure the links would work directly, without needing to remember a correct url. 
```suggestion
- UK version: [http://localhost.bbc.co.uk:7080/news/articles/c0000000001o](http://localhost.bbc.co.uk:7080/news/articles/c0000000001o)
```

File: README.md [hunk 3]
```diff
@@ -77,6 +77,13 @@ To run LIVE bundles on localhost:
...
+- International version: [http://localhost.bbc.com:7080](http://localhost.bbc.com:7080)
```
Comment (sareh) [inline README.md]:
```suggestion
- International version: [http://localhost.bbc.com:7080/news/articles/c0000000001o](http://localhost.bbc.com:7080/news/articles/c0000000001o)
```

**Review by sareh — COMMENTED** [review]
Thanks for adding this. Just a couple of suggestions added to make steps clearer for new users of the repo.

**Review by chris-hinds — APPROVED** [review]
👍

**Review by 12 — APPROVED** [review]

**Comment by staylos92** [comment]
No formal testing required for this

**Pull request merged**